sdk/messaging/azservicebus/README.md

```diff
@@ -349,6 +335,5 @@ If you'd like to contribute to this library, please read the [contributing guide
 [godoc_processor]: https://godoc.org/github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus/#Processor
 [godoc_processor_deadlettermessage]: https://godoc.org/github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus/#Processor.DeadLetterMessage
 [godoc_newsender]: https://godoc.org/github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus/#Client.NewSender
-[godoc_newreceiver_queue]: https://godoc.org/github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus/#Client.NewReceiverWithQueue
-[godoc_newreceiver_subscription]: https://godoc.org/github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus/#Client.NewReceiverWithQueue
--->
+[godoc_newreceiver_queue]: https://godoc.org/github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus/#Client.NewReceiverForQueue
```

Comment:
Really should be using pkg.go.dev instead of godoc.org however the latter redirects to the former so it's not urgent.

Reply:
Dang it, I should not have set that to auto-merge.

Reply:
https://github.com/Azure/azure-sdk-for-go/pull/15721